### PR TITLE
Adding new locales to android-l10n

### PR DESCRIPTION
**Pull request by Delphine**

### CODE REVIEW

**Comment by Delphine** [comment]
After a few back and forths... this is ready for review 😁

File: src/products/android-l10n/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# L10n and android-l10n
```
Comment (Pike) [inline src/products/android-l10n/README.md:1]: Does this need to be hooked up in `SUMMARY.md` ?

File: src/products/android-l10n/adding_new_locales.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Edit the l10n.toml file(s)
+
+All android-l10n products live within the [android-l10n project](https://github.com/mozilla-l10n/android-l10n/). Depending on the product that requires new locales to be added, the corresponding l10n.toml file will have to be edited by adding the new locale(s) code(s) in it.
+
+The l10n.toml files live in their corresponding project folder, located here for:
+* Fenix: https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-mobile/fenix/l10n.toml
+* Android-Components: https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-mobile/android-components/l10n.toml
+* Firefox for Fire TV: https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-mobile/firefox-tv/l10n.toml
+* Firefox Reality: https://github.com/mozilla-l10n/android-l10n/blob/master/MozillaReality/FirefoxReality/l10n.toml
+* Lockwise for Android: https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-lockwise/lockwise-android/l10n.toml
```
Comment (Pike) [inline src/products/android-l10n/adding_new_locales.md]: I'd suggest to shorten these links.

File: src/products/android-l10n/adding_new_locales.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Identify the `locales` section, add the desired new locale(s) code(s) in this  list. With Atom and the Sort Lines package installed, you can press `F5` to make sure that the list is in **alphabetical order**.
+
+Save your edited file, then commit and push from your branch to the android-l10n repository.
+
+Note that all products except Firefox for Fire TV share strings with Android Components, so you will also have to edit the Android Components l10n.toml file and add the new locale(s) there.
+
+## Add new locales to Pontoon
+
+Once the patch has landed, the new locale(s) have to be added in Pontoon as well. The steps to follow can be found in the existing [Pontoon documentation here](https://github.com/mozilla-l10n/documentation/blob/master/src/tools/pontoon/adding_new_locale.md).
```
Comment (Pike) [inline src/products/android-l10n/adding_new_locales.md]: Does this link work after mdbook?

**Review by Pike — COMMENTED** [review]
Technically, this is right. But there's stuff around how the docs work that I'm unsure about.

File: src/products/android-l10n/adding_new_locales.md (hunk 3 above)
Comment (flodolo) [reply src/products/android-l10n/adding_new_locales.md]: This should be a relative link.

File: src/products/android-l10n/README.md (hunk 1 above)
Comment (Delphine) [reply src/products/android-l10n/README.md:1]: Good catch. Yes it should

**Comment by Delphine** [comment]
I think the latest commit should have addressed the issues.

**Comment by Delphine** [comment]
I don't understand what Travis is telling me about mdbooks, so won't  be able to fix this now.

File: src/SUMMARY.md [hunk 4]
```diff
@@ -4,6 +4,8 @@
 
 ## Products
 
+* [Android-l10n](products/android_l10n)
```
Comment (flodolo) [inline src/SUMMARY.md]: Link to the README (I assume that's what breaking the book build)

File: src/products/android-l10n/adding_new_locales.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Identify the `locales` section, add the desired new locale(s) code(s) in this  list. With Atom and the Sort Lines package installed, you can press `F5` to make sure that the list is in **alphabetical order**.
+
+Save your edited file, then commit and push from your branch to the android-l10n repository.
+
+Note that all products except Firefox for Fire TV share strings with Android Components, so you will also have to edit the Android Components l10n.toml file and add the new locale(s) there.
+
+## Add new locales to Pontoon
+
+Once the patch has landed, the new locale(s) have to be added in Pontoon as well. The steps to follow can be found in the existing [Pontoon documentation here](tools/pontoon/adding_new_locale.md).
```
Comment (flodolo) [inline src/products/android-l10n/adding_new_locales.md]: Still not a relative link (i.e. relative to the current page).

Current page: products/android-l10n/adding_new_locales.md
Link: tools/pontoon/adding_new_locale.md
Relative link: ../../tools/pontoon/adding_new_locale.md

Comment (Delphine) [reply src/products/android-l10n/adding_new_locales.md]: Oh ok wasn't too sure what it had to be. I'll update that, thanks

**Comment by Delphine** [comment]
Hopefully this is good to go now... 🥺

File: src/products/android-l10n/adding_new_locales.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+Identify the `locales` section, add the desired new locale(s) code(s) in this  list. With Atom and the Sort Lines package installed, you can press `F5` to make sure that the list is in **alphabetical order**.
+
+Save your edited file, then commit and push from your branch to the android-l10n repository.
+
+Note that all products except Firefox for Fire TV share strings with Android Components, so you will also have to edit the Android Components l10n.toml file and add the new locale(s) there.
+
+## Add new locales to Pontoon
+
+Once the patch has landed, the new locale(s) have to be added in Pontoon as well. The steps to follow can be found in the existing [Pontoon documentation here](../../tools/pontoon/adding_new_locale.md).
```
Comment (Pike) [inline src/products/android-l10n/adding_new_locales.md]: `.md` or `.html`? No idea if mdbook fixes up the extensions.

**Review by Pike — APPROVED** [review]

File: src/products/android-l10n/adding_new_locales.md (hunk 6 above)
Comment (Delphine) [reply src/products/android-l10n/adding_new_locales.md]: That's what I've seen in other links, but I may be mistaken?

Comment (flodolo) [reply src/products/android-l10n/adding_new_locales.md]: `.md` is correct

File: src/SUMMARY.md [hunk 7]
```diff
@@ -4,6 +4,8 @@
 
 ## Products
 
+* [Android-l10n](products/android_l10n/README.md)
+    * [Adding new locales to specific projects](products/android_l10n/adding_new_locales.md).
```
Comment (flodolo) [inline src/SUMMARY.md:8]: Might be good to shorten this one, since it's in the sidebar. "Adding new locales" could be enough?

**Pull request merged**